nixos/tests/grafana/provision/default.nix

```diff
@@ -192,14 +168,6 @@ in {
             )
 
 
-
-    with subtest(f"Successful notifiers provision with {nodeOld[0]}"):
```

Comment:
We haven't actually deprecated notifiers provisioning completely, we just put up a warning about it. Please don't remove the test. We can still test the behaviour if we move the notifiers provisoning into `nodeNix`.
Also, since we removed the `nodeOld`, we can now write all of the tests inside of the `for nodeInfo in [nodeNix, nodeYaml]` loop. That would look quite a bit cleaner.